src/components/dictionaryConcepts/components/ActionButtons.jsx

```diff
@@ -15,28 +16,33 @@ const ActionButtons = ({
   if (actionButtons === true) {
     showExtra = true;
   }
-
   return (
-    !retired && <React.Fragment>
-      {showExtra && (
-        <React.Fragment>
-          <Link
-            className="edit-button-link btn btn-sm mb-1 actionButtons"
-            to={`/edit/${concept_class}/${id}${dictionaryPathName}`}
+    !retired && (
+      <React.Fragment>
+        {showExtra && (
+          <React.Fragment>
+            <Link
+              className="edit-button-link btn btn-sm mb-1 actionButtons"
+              to={`/edit/${concept_class}/${id}${dictionaryPathName}`}
+            >
+              Edit
```

Comment:
How are these changes related to the getting rid of the remove button?

Reply:
It was a linting fix. Wrapped the whole`React.Fragment` tag, which included the remove button, in brackets.

Reply:
I removed the unnecessary changes here